src/map/cashshop.c

```diff
@@ -236,6 +567,7 @@ bool cashshop_buylist( struct map_session_data* sd, uint32 kafrapoints, int n, u
 	for( i = 0; i < n; ++i ){
 		unsigned short nameid = *( item_list + i * 5 );
 		uint32 quantity = *( item_list + i * 5 + 2 );
+		uint16 tab = *(item_list + i * 5 + 4);
```

Comment:
No it was always there to begin with(`i * 5`). Only mistake here is that I used uint16 instead of uint8